.github/workflows/ci.yaml

```diff
@@ -13,14 +13,32 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version-file: go.mod
-          cache: true
-      - run: make setup
+      - uses: ./.github/actions/aqua
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
       - run: make lint
       - run: make check-generate
       - run: make test
-  build-image:
-    name: Build Container Image
+  dry-run:
+    name: Dry-run release
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v3
-      - run: docker build -t quay.io/cybozu/pod-security-admission:latest .
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
```

Comment:
It appears that setup-qemu-action and setup-buildx-action have not been released in v1 for some time.
The experimental features of docker are still valid in v1 and it seems to be able to push images with goreleaser without problems.
I'll leave it to you to decide if you want to go to v2.